src/jdk.compiler/share/classes/com/sun/tools/javac/code/Symbol.java

```diff
@@ -1783,6 +1784,11 @@ public void setData(Object data) {
         public <R, P> R accept(Symbol.Visitor<R, P> v, P p) {
             return v.visitVarSymbol(this, p);
         }
+
+        @Override @DefinedBy(Api.LANGUAGE_MODEL)
```

Comment:
It's not incorrect to have this override, but I don't think it is necessary now either with the update to the default method.